app/src/main/java/org/sagebionetworks/research/mpower/tracking/view_model/TrackingActiveTaskViewModel.java

```diff
@@ -0,0 +1,145 @@
+package org.sagebionetworks.research.mpower.tracking.view_model;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.Transformations;
+import android.arch.lifecycle.ViewModel;
+import android.support.annotation.NonNull;
+
+import org.sagebionetworks.research.mpower.tracking.model.TrackingItem;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingSection;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingStep;
+import org.sagebionetworks.research.mpower.tracking.model.TrackingStepView;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public abstract class TrackingActiveTaskViewModel<ConfigType extends TrackingItemConfig, LogType extends TrackingItemLog>
```

Comment:
I'm used to the term "active" meaning sensor related. What does it mean in this case?